agents/model.py

```diff
@@ -115,11 +115,26 @@ def point_towards(self, other_x, other_y):
             if (self.y - other_y) > 0:
                 self.direction = 360 - self.direction
 
-    def forward(self):
-        self.x += math.cos(math.radians(self.direction)) * self.speed
-        self.y += math.sin(math.radians(self.direction)) * self.speed
+    def forward(self, distance=None):
+        if distance == None:
+            distance = self.speed
+        self.x += math.cos(math.radians(self.direction)) * distance
+        self.y += math.sin(math.radians(self.direction)) * distance
         self.__post_move()
 
+    def backward(self, distance=None):
+        if distance == None:
+            distance = self.speed
+        self.x -= math.cos(math.radians(self.direction)) * distance
+        self.y -= math.sin(math.radians(self.direction)) * distance
+        self.__post_move()
+
+    def agent.left(self, degrees):
+        self.direction += degrees
```

Comment:
shouldn't it just be `def left(self, degrees)`?

Reply:
Oh yeah.